docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -2681,10 +2738,10 @@ Status Codes:
 
 JSON Parameters:
 
-- **name** - The new network's name. this is a mandatory field
-- **driver** - Name of the network driver to use. Defaults to `bridge` driver
-- **options** - Network specific options to be used by the drivers
-- **check_duplicate** - Requests daemon to check for networks with same name
+- **Name** - The new network's name. this is a mandatory field
+- **Driver** - Name of the network driver to use. Defaults to `bridge` driver
+- **Options** - Network specific options to be used by the drivers
+- **CheckDuplicate** - Requests daemon to check for networks with same name
```

Comment:
I noticed `IPAM` was missing here, but didn't know how that should be used; https://github.com/docker/docker/pull/17536/files#diff-8bd32afa592e764a30005657f5494613R360